third_party/libpg_query/grammar/statements/variable_show.y

```diff
@@ -64,3 +64,8 @@ var_name:	ColId								{ $$ = $1; }
 			| var_name '.' ColId
 				{ $$ = psprintf("%s.%s", $1, $3); }
 		;
+
+var_name_describe:	ColId								{ $$ = psprintf("\"%s\"", $1); }
```

Comment:
That was my initial way to fix it, but I was getting weird results where my PGRangeVar was not getting initialized properly.

Reply:
Hm, could you share the code that you tried?

Reply:
Ah nvm, now it works, was probably related to the changes my use of Bison 3.8.4 caused (which have been reverted)

Reply:
Since the pragma info takes a list of Values, I do have to give the returned qualified name the same treatment as the parser does with these changes, because I need to separate the schema from the table later